operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/Utils.java

```diff
@@ -97,19 +97,8 @@ public static boolean debugThreadPool() {
     return getBooleanFromSystemPropsOrDefault(DEBUG_THREAD_POOL_ENV_KEY, false);
   }
 
-  static boolean getBooleanFromSystemPropsOrDefault(String propertyName, boolean defaultValue) {
-    var property = System.getProperty(propertyName);
-    if (property == null) {
-      return defaultValue;
-    } else {
-      property = property.trim().toLowerCase();
-      return switch (property) {
-        case "true" -> true;
-        case "false" -> false;
-        default -> defaultValue;
-      };
-    }
-  }
+  static boolean getBooleanFromSystemPropsOrDefault(String propertyName,
+      boolean defaultValue) {var property=System.getProperty(propertyName);if(property==null){return defaultValue;}else{property=property.trim().toLowerCase();return switch(property){case"true"->true;case"false"->false;default->defaultValue;};}}
```

Comment:
Hmm, that doesn't seem right

Reply:
wow, interesting, not sure, will take a look a look later, this might be a bug in spotless

Reply:
Yes, there seems something wrong with 2.44